actionpack/test/dispatch/routing_test.rb

```diff
@@ -2795,12 +2795,10 @@ def test_except_option_should_override_scope
 
   def test_except_option_should_not_inherit
     draw do
-      scope except: :index do
```

Comment:
Because of this change this test is not testing what it should be testing. Maybe we should change the `:index` to `:delete`?